[2025-09-03T20:17:14.631+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: continuous_streaming_consumer.continuous_producer_consumer scheduled__2025-09-03T20:15:00+00:00 [queued]>
[2025-09-03T20:17:14.635+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: continuous_streaming_consumer.continuous_producer_consumer scheduled__2025-09-03T20:15:00+00:00 [queued]>
[2025-09-03T20:17:14.636+0000] {taskinstance.py:1279} INFO - 
--------------------------------------------------------------------------------
[2025-09-03T20:17:14.636+0000] {taskinstance.py:1280} INFO - Starting attempt 1 of 1000
[2025-09-03T20:17:14.636+0000] {taskinstance.py:1281} INFO - 
--------------------------------------------------------------------------------
[2025-09-03T20:17:14.642+0000] {taskinstance.py:1300} INFO - Executing <Task(PythonOperator): continuous_producer_consumer> on 2025-09-03 20:15:00+00:00
[2025-09-03T20:17:14.645+0000] {standard_task_runner.py:55} INFO - Started process 352 to run task
[2025-09-03T20:17:14.646+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'continuous_streaming_consumer', 'continuous_producer_consumer', 'scheduled__2025-09-03T20:15:00+00:00', '--job-id', '6', '--raw', '--subdir', 'DAGS_FOLDER/dag_continuous_consumer.py', '--cfg-path', '/tmp/tmp_xmytwxv']
[2025-09-03T20:17:14.648+0000] {standard_task_runner.py:83} INFO - Job 6: Subtask continuous_producer_consumer
[2025-09-03T20:17:14.658+0000] {logging_mixin.py:137} WARNING - /usr/local/lib/python3.9/site-packages/***/settings.py:249 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-09-03T20:17:14.671+0000] {logging_mixin.py:137} WARNING - /usr/local/lib/python3.9/site-packages/***/utils/sqlalchemy.py:124 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-09-03T20:17:14.692+0000] {task_command.py:388} INFO - Running <TaskInstance: continuous_streaming_consumer.continuous_producer_consumer scheduled__2025-09-03T20:15:00+00:00 [running]> on host 15e5c1330dfb
[2025-09-03T20:17:14.733+0000] {taskinstance.py:1507} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=Klaus_Rezende
AIRFLOW_CTX_DAG_ID=continuous_streaming_consumer
AIRFLOW_CTX_TASK_ID=continuous_producer_consumer
AIRFLOW_CTX_EXECUTION_DATE=2025-09-03T20:15:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2025-09-03T20:15:00+00:00
[2025-09-03T20:17:14.737+0000] {conn.py:380} INFO - <BrokerConnection node_id=bootstrap-0 host=kafka:29092 <connecting> [IPv4 ('172.16.240.5', 29092)]>: connecting to kafka:29092 [('172.16.240.5', 29092) IPv4]
[2025-09-03T20:17:14.738+0000] {conn.py:1205} INFO - Probing node bootstrap-0 broker version
[2025-09-03T20:17:14.738+0000] {conn.py:410} INFO - <BrokerConnection node_id=bootstrap-0 host=kafka:29092 <connecting> [IPv4 ('172.16.240.5', 29092)]>: Connection complete.
[2025-09-03T20:17:14.741+0000] {conn.py:380} INFO - <BrokerConnection node_id=bootstrap-0 host=kafka:29092 <connecting> [IPv4 ('172.16.240.5', 29092)]>: connecting to kafka:29092 [('172.16.240.5', 29092) IPv4]
[2025-09-03T20:17:14.742+0000] {conn.py:1205} INFO - Probing node bootstrap-0 broker version
[2025-09-03T20:17:14.742+0000] {conn.py:410} INFO - <BrokerConnection node_id=bootstrap-0 host=kafka:29092 <connecting> [IPv4 ('172.16.240.5', 29092)]>: Connection complete.
[2025-09-03T20:17:14.840+0000] {conn.py:1267} INFO - Broker version identified as 2.5.0
[2025-09-03T20:17:14.840+0000] {conn.py:1268} INFO - Set configuration api_version=(2, 5, 0) to skip auto check_version requests on startup
[2025-09-03T20:17:14.844+0000] {conn.py:1267} INFO - Broker version identified as 2.5.0
[2025-09-03T20:17:14.844+0000] {conn.py:1268} INFO - Set configuration api_version=(2, 5, 0) to skip auto check_version requests on startup
[2025-09-03T20:17:14.845+0000] {consumer.py:118} WARNING - group_id is None: disabling auto-commit.
[2025-09-03T20:17:14.845+0000] {subscription_state.py:171} INFO - Updating subscribed topics to: ('streaming_data',)
[2025-09-03T20:17:14.846+0000] {subscription_state.py:257} INFO - Updated partition assignment: [('streaming_data', 0)]
[2025-09-03T20:17:14.846+0000] {conn.py:380} INFO - <BrokerConnection node_id=1 host=kafka:29092 <connecting> [IPv4 ('172.16.240.5', 29092)]>: connecting to kafka:29092 [('172.16.240.5', 29092) IPv4]
[2025-09-03T20:17:14.847+0000] {conn.py:410} INFO - <BrokerConnection node_id=1 host=kafka:29092 <connecting> [IPv4 ('172.16.240.5', 29092)]>: Connection complete.
[2025-09-03T20:17:14.847+0000] {conn.py:919} INFO - <BrokerConnection node_id=bootstrap-0 host=kafka:29092 <connected> [IPv4 ('172.16.240.5', 29092)]>: Closing connection. 
[2025-09-03T20:17:14.934+0000] {conn.py:380} INFO - <BrokerConnection node_id=1 host=kafka:29092 <connecting> [IPv4 ('172.16.240.5', 29092)]>: connecting to kafka:29092 [('172.16.240.5', 29092) IPv4]
[2025-09-03T20:17:14.934+0000] {conn.py:410} INFO - <BrokerConnection node_id=1 host=kafka:29092 <connecting> [IPv4 ('172.16.240.5', 29092)]>: Connection complete.
[2025-09-03T20:17:14.934+0000] {conn.py:919} INFO - <BrokerConnection node_id=bootstrap-0 host=kafka:29092 <connected> [IPv4 ('172.16.240.5', 29092)]>: Closing connection. 
[2025-09-03T20:17:14.936+0000] {logging_mixin.py:137} INFO - Produced post 1: sunt aut facere repellat provi...
[2025-09-03T20:17:24.953+0000] {logging_mixin.py:137} INFO - Produced post 2: qui est esse...
[2025-09-03T20:17:24.956+0000] {logging_mixin.py:137} INFO - Consumed: qui est esse...
[2025-09-03T20:17:34.951+0000] {logging_mixin.py:137} INFO - Produced post 3: ea molestias quasi exercitatio...
[2025-09-03T20:17:44.963+0000] {logging_mixin.py:137} INFO - Produced post 4: eum et est occaecati...
[2025-09-03T20:17:54.966+0000] {logging_mixin.py:137} INFO - Produced post 5: nesciunt quas odio...
[2025-09-03T20:18:55.587+0000] {logging_mixin.py:137} INFO - Produced post 6: dolorem eum magni eos aperiam ...
[2025-09-03T20:18:14.988+0000] {logging_mixin.py:137} INFO - Produced post 7: magnam facilis autem...
[2025-09-03T20:18:25.012+0000] {logging_mixin.py:137} INFO - Produced post 8: dolorem dolore est ipsam...
[2025-09-03T20:18:35.010+0000] {logging_mixin.py:137} INFO - Produced post 9: nesciunt iure omnis dolorem te...
[2025-09-03T20:18:45.032+0000] {logging_mixin.py:137} INFO - Produced post 10: optio molestias id quia eum...
[2025-09-03T20:18:55.032+0000] {logging_mixin.py:137} INFO - Produced post 11: et ea vero quia laudantium aut...
[2025-09-03T20:19:05.041+0000] {logging_mixin.py:137} INFO - Produced post 12: in quibusdam tempore odit est ...
[2025-09-03T20:19:15.056+0000] {logging_mixin.py:137} INFO - Produced post 13: dolorum ut in voluptas molliti...
[2025-09-03T20:20:14.351+0000] {conn.py:919} INFO - <BrokerConnection node_id=1 host=kafka:29092 <connected> [IPv4 ('172.16.240.5', 29092)]>: Closing connection. KafkaConnectionError: Socket EVENT_READ without in-flight-requests
[2025-09-03T20:20:14.351+0000] {client_async.py:331} WARNING - Node 1 connection failed -- refreshing metadata
[2025-09-03T20:20:14.402+0000] {conn.py:380} INFO - <BrokerConnection node_id=1 host=kafka:29092 <connecting> [IPv4 ('172.16.240.5', 29092)]>: connecting to kafka:29092 [('172.16.240.5', 29092) IPv4]
[2025-09-03T20:20:14.403+0000] {conn.py:418} ERROR - Connect attempt to <BrokerConnection node_id=1 host=kafka:29092 <connecting> [IPv4 ('172.16.240.5', 29092)]> returned error 111. Disconnecting.
[2025-09-03T20:20:14.403+0000] {conn.py:919} INFO - <BrokerConnection node_id=1 host=kafka:29092 <connecting> [IPv4 ('172.16.240.5', 29092)]>: Closing connection. KafkaConnectionError: 111 ECONNREFUSED
[2025-09-03T20:20:14.403+0000] {client_async.py:331} WARNING - Node 1 connection failed -- refreshing metadata
[2025-09-03T20:20:14.605+0000] {conn.py:380} INFO - <BrokerConnection node_id=1 host=kafka:29092 <connecting> [IPv4 ('172.16.240.5', 29092)]>: connecting to kafka:29092 [('172.16.240.5', 29092) IPv4]
[2025-09-03T20:20:14.605+0000] {conn.py:418} ERROR - Connect attempt to <BrokerConnection node_id=1 host=kafka:29092 <connecting> [IPv4 ('172.16.240.5', 29092)]> returned error 111. Disconnecting.
[2025-09-03T20:20:14.605+0000] {conn.py:919} INFO - <BrokerConnection node_id=1 host=kafka:29092 <connecting> [IPv4 ('172.16.240.5', 29092)]>: Closing connection. KafkaConnectionError: 111 ECONNREFUSED
[2025-09-03T20:20:14.605+0000] {client_async.py:331} WARNING - Node 1 connection failed -- refreshing metadata
[2025-09-03T20:20:14.908+0000] {conn.py:380} INFO - <BrokerConnection node_id=1 host=kafka:29092 <connecting> [IPv4 ('172.16.240.5', 29092)]>: connecting to kafka:29092 [('172.16.240.5', 29092) IPv4]
[2025-09-03T20:20:14.908+0000] {conn.py:418} ERROR - Connect attempt to <BrokerConnection node_id=1 host=kafka:29092 <connecting> [IPv4 ('172.16.240.5', 29092)]> returned error 111. Disconnecting.
[2025-09-03T20:20:14.908+0000] {conn.py:919} INFO - <BrokerConnection node_id=1 host=kafka:29092 <connecting> [IPv4 ('172.16.240.5', 29092)]>: Closing connection. KafkaConnectionError: 111 ECONNREFUSED
[2025-09-03T20:20:14.908+0000] {client_async.py:331} WARNING - Node 1 connection failed -- refreshing metadata
[2025-09-03T20:20:15.462+0000] {conn.py:380} INFO - <BrokerConnection node_id=1 host=kafka:29092 <connecting> [IPv4 ('172.16.240.5', 29092)]>: connecting to kafka:29092 [('172.16.240.5', 29092) IPv4]
